src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/ExtensionElement.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Configuration;
+
+namespace CoreWCF.Configuration
+{
+    internal class ExtensionElement : ConfigurationElement
+    {
```

Comment:
This is public in .NET Framework, was there a reason to make this internal?

Reply:
I did not see a need to make it public.  I added it to support ServiceModelExtensionCollectionElement.GetExtensionType().  Since support was not being added for separate ExtensionElements I did not see any value in making public.  I have no compelling reason to keep internal.

Reply:
That makes sense. If we don't support extension elements, then it makes sense to keep it internal.